README.md

```diff
@@ -14,6 +22,18 @@ $ anonymousface < input.jpg > output.jpg
 go install github.com/mattn/anonymousface@latest
 ```
 
+### Using make file
+
+```bash
+git clone https://github.com/mattn/anonymousface.git && cd anonymousface
+```
+
+and then
+
+```bash
+make
```

Comment:
Makefile is not provided for the users. This is for maintener. (me)

Reply:
alright, so should I add build instruction using `go install` as the current instruction has some issue?